dashboards/flink/flink-task-manager-prometheus.json

```diff
@@ -32,25 +33,32 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "flink_taskmanager_Status_JVM_CPU_Load{${Cluster},${Location},${Namespace}}"
+                  "prometheusQuery": "flink_taskmanager_Status_JVM_CPU_Load{${Cluster},${Location},${Namespace}}",
+                  "unitOverride": ""
                 }
               }
             ],
+            "thresholds": [],
             "timeshiftDuration": "0s",
             "yAxis": {
+              "label": "",
               "scale": "LINEAR"
             }
-          }
-        },
-        "width": 4,
-        "xPos": 0,
-        "yPos": 0
+          },
+          "computedId": "f7cd67cc-576b-4c27-8f04-42f613e4179c"
```

Comment:
I'm not sure what this field is. It's present in all the charts. I'm not able to create a dashboard with this field. Is this required here?

Reply:
I'm going to get rid of it. The appearance of this field coincided with a bug where I was unable to make changes to the dashboard via the JSON editor, and its removal appears to have solved that bug.